src/pages/search/index.tsx

```diff
@@ -226,7 +226,10 @@ const Search = () => {
   };
 
   return (
-    <Layout title={t("Law and Policy Search")} heading={t("Law and Policy Search")}>
+    <Layout
+      title={t("Law and Policy Search")}
+      description="Quickly and easily search through the complete text of thousands of climate change law and policy documents from every country."
```

Comment:
This seems a bit off... there aren't thousands of law/policy documents from every country, and we don't have documents for all countries, so whichever way I parse this it's a lie :grin: